src/components/detail/PostFooter/Comment/Comment.tsx

```diff
@@ -0,0 +1,46 @@
+import * as S from "./style";
+import ProfileImage from "../../../common/ProfileImage";
+import theme from "../../../../styles/theme";
+
+interface CommentInterface {
+  comment: string;
+  author: string;
+  updatedAt: string;
+}
+
+const deleteComment = () => {
+  if (window.confirm("댓글을 삭제하시겠습니까?")) {
+    alert("삭제 구현중!!");
+  }
+};
+
+const Comment: React.FC<CommentInterface> = ({
+  comment,
+  author,
+  updatedAt
+}) => {
+  return (
+    <div>
+      <S.FlexBetween style={{ alignItems: "center" }}>
+        <S.CenterAlignItemSpan>
+          <ProfileImage block={false} size={"md"}></ProfileImage>
```

Comment:
- 문자열값은 중괄호 없이 바로 넘길 수 있어요!
- ProfileImage 컴포넌트가 children을 사용하지 않아서 바로 태그를 닫을 수 있어요!
```suggestion
          <ProfileImage block={false} size="md" />
```

Reply:
오호! 프로필 이미지 자주 쓰는데 꿀팁 감사합니다!! 👍👍👍